WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/Views/MigrationHeaderView.swift

```diff
@@ -50,19 +51,12 @@ final class MigrationHeaderView: UIView {
         labelsStackView.translatesAutoresizingMaskIntoConstraints = false
         labelsStackView.spacing = Constants.labelsSpacing
         let mainStackView = verticalStackView(arrangedSubviews: [imageView, labelsStackView])
-        mainStackView.spacing = Constants.spacing
+        mainStackView.setCustomSpacing(Constants.spacing, after: imageView)
         mainStackView.alignment = .leading
         mainStackView.translatesAutoresizingMaskIntoConstraints = false
         self.addSubview(mainStackView)
+        pinSubviewToAllEdges(mainStackView)
         configureAppearance()
-        // Set constraints
-        NSLayoutConstraint.activate([
-            labelsStackView.widthAnchor.constraint(equalTo: mainStackView.widthAnchor),
-            mainStackView.topAnchor.constraint(equalTo: layoutMarginsGuide.topAnchor),
-            mainStackView.leadingAnchor.constraint(equalTo: readableContentGuide.leadingAnchor),
-            mainStackView.trailingAnchor.constraint(equalTo: readableContentGuide.trailingAnchor),
-            mainStackView.bottomAnchor.constraint(equalTo: layoutMarginsGuide.bottomAnchor)
-        ])
```

Comment:
The existing code was laying out the `mainStackView` leading and trailing anchors to `readableContentGuide`, which is different than `pinSubviewToAllEdges` constraints.

`layoutMarginsGuide` and `readableContentGuide` are the same for compact screens like iPhone but they different on iPad. Here is a comparison:

| layoutMarginsGuide | readableContentGuide |
| -------------------- | ----------------------- |
| ![](https://user-images.githubusercontent.com/9609223/201940753-988f0bb6-89ac-4b23-8862-caf2b3d2145d.png) | ![](https://user-images.githubusercontent.com/9609223/201941405-15a5722a-c001-43cd-ad2b-1fd6910c015e.png) |

👉 [Link](https://manasaprema04.medium.com/readablecontentguide-in-swift-e54243d1184d) for more info on `readableContentGuide`